docs/pod_network_scenarios.md

```diff
@@ -1,3 +1,5 @@
+## Pod network Scenarios
```

Comment:
Let's add pod network scenarios to the list: https://github.com/redhat-chaos/krkn/tree/main#kubernetesopenshift-chaos-scenarios-supported

Reply:
done.